libfoundation/src/foundation-value.cpp

```diff
@@ -802,7 +802,7 @@ bool __MCValueInitialize(void)
 
 void __MCValueFinalize(void)
 {
-    for(uindex_t i = 0; i < sizeof(s_value_pools) / sizeof(s_value_pools[0]); i++)
+    for(index_t i = 0; i < sizeof(s_value_pools) / sizeof(s_value_pools[0]); i++)
```

Comment:
Well clearly that code is total rubbish! My fault...

Ages and ages ago there was a problem (which we never found the cause of) on Linux with regards the static init of s_value_pools; so I changed it to a dynamically allocated array. Obviously I neglected to update *that* code!